.prettierignore

```diff
@@ -0,0 +1,7 @@
+*.marko.js
+static/
+dist/
+docs/
+_cdn/
+demo/style.less
+src/components/ebay-video/style.less
```

Comment:
Put a newline at the end of the file.
Just curious why is ebay-video style ignored?

Reply:
I was trying not to change the files for this PR, because there weren't any changes in that file related to this PR, other than styling. For some reason that file kept getting picked up and prettier was running on it

Reply:
Well there is a prettier ignore thing you added in the top of the file, so maybe thats why?
Regardless either we have ignore on those files or in here but no reason to have it in both places. 

And I think it's fine to have those files changed